Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -59,11 +99,15 @@ Manual Verification of Setup
 
  #. Validate that the connection to the remote cluster could be established.
     You will see a log message like this in the ``cilium-agent`` logs for each
-    remote cluster::
+    remote cluster
+
+    .. code-block:: shell-session
 
        level=info msg="Connection to remote cluster established"
 
-    If the connection failed, you will see a warning like this::
+    If the connection failed, you will see a warning like this
+
+    .. code-block:: shell-session
```

Comment:
These remaining blocks in this patch are not shell session output. Shell session highlights the `$` / `#` at start of line and colorizes the commands vs. the output. This is all just output of commands, so `::` is the best option for generic output with no special syntax highlighting.